packages/@react-spectrum/list/src/styles.css

```diff
@@ -131,10 +131,106 @@
   }
 }
 
-.react-spectrum-ListView--quiet {
+.react-spectrum-ListView.react-spectrum-ListView--quiet {
   background-color: var(--spectrum-alias-background-color-transparent);
   border-width: 0;
   border-radius: 0;
+  &:not(.react-spectrum-ListView--multipleSelection) {
```

Comment:
This is... quite the CSS. Not sure I follow completely, but there's no way this can be simplified?

Reply:
I see a place i can simplify a little 👍🏻